django/db/models/base.py

```diff
@@ -1443,7 +1443,7 @@ def _check_single_primary_key(cls):
         if sum(1 for f in cls._meta.local_fields if f.primary_key) > 1:
             errors.append(
                 checks.Error(
-                    "Model can not contain more than one 'primary_key' field.",
+                    "Model cannot contain more than one 'primary_key' field.",
```

Comment:
The model cannot have more than one field with 'primary_key=True'.

Reply:
Good. Yes. (Programmers! 🙂)